mp/game/momentum/resource/ui/SettingsPanel_HudSettings.res

```diff
@@ -121,10 +151,40 @@
         "auto_tall_tocontents" "1"
         "use_proportional_insets"		"0"
         "Default"		"0"
-        "cvar_name"		"mom_hud_speedometer_hvel"
-        "cvar_value"		"0"
+        "cvar_name"		"mom_hud_speedometer_showenterspeed"
+        "cvar_value"		"1"
```

Comment:
Same here, I would remove cvar_value since you're already touching this in this PR, but no others.